### PR TITLE
Migrate to Swift4.2

### PR DESCRIPTION
**Pull request by baquer**
Migrate Project To Swift 4.2

### CODE REVIEW

**Comment by baquer** [comment]
@jogendra , @chashmeetsingh  Review the PR.

File: FOSSAsia.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -604,7 +604,6 @@
 				2824FD6B1C6DBF9A00F783E1 /* More.storyboard in Resources */,
 				285F89B21C8149420033D090 /* Favorites.storyboard in Resources */,
 				285F1B711C5AF7B100A7E0DD /* LaunchScreen.storyboard in Resources */,
-				186729E720F762EC00C3AA45 /* Profile.storyboard in Resources */,
```
Comment (jogendra) [inline FOSSAsia.xcodeproj/project.pbxproj:607]: Why did you remove reference of `Profile` storyboard?

Comment (baquer) [reply FOSSAsia.xcodeproj/project.pbxproj:607]: Oop's ,will fix this , and some errors are occurring too while building the project ,so will fix all these issues and make a commit soon.

Comment (baquer) [reply FOSSAsia.xcodeproj/project.pbxproj:607]: @jogendra , I checked it twice , the reference of Profile storyboard gets automatically removed after migrating to swift4.2 , here is the screenshot Don't know why this is happening? . And the app is working fine.
screenshot 2018-11-02 at 10 22 25 pm

**Comment by jogendra** [comment]
I am closing this PR, please make a PR for migration to Swift 5!
